packages/lesswrong/lib/types/collectionTypes.ts

```diff
@@ -188,7 +188,7 @@ interface VoteableTypeClient extends VoteableType {
   currentUserExtendedVote?: any,
 }
 
-interface DbVoteableType extends VoteableType, DbObject {
+interface DbVoteableType extends VoteableType, DbObject, HasUserIdType {
```

Comment:
Removing `userId` from the fragments required moving `HasUserIdType` from `VoteableType` to just `DbVoteableType`. This is the cause of all the other type changes, all of which were just due to using an overly general type and weren't real issues